.changelog/90358f5074924d8d8b736922ebb7e586.json

```diff
@@ -0,0 +1,330 @@
+{
+    "id": "90358f50-7492-4d8d-8b73-6922ebb7e586",
```

Comment:
You probably don't actually want to include this changelog note since this will be part of the release. Otherwise there will be multiple changelog entries with the same information.